content/_plugins.md

```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
+ "temp_blueprint_id": "bbf4b172-3b21-460c-aed8-4f55b55f3b4f",
+ "created_by": "admin",
+ "deployments_to_update": [
+    "dd7b03fb3-8f92-404c-8ddd-d81bd6a2bc9b"
+ ],
+ "resource_availability": "tenant",
+ "id": "ce63a9d6-726b-4ff2-a267-f953af5dc32d",
+ "execution_id": "8ec77e89-b418-4472-9b44-fe9d7a7fe163"
+}
+
+```
+
+`PUT "<manager-ip>/api/v3.1/plugins-updates/<blueprint-id>/update/initiate"`
+
+Update the plugins for that blueprint's deployments. **Supported for Cloudify Manager 5.0 and above.**
+
+### URI Parameters
+* `blueprint-id`: The id of the blueprint to update its deployments.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`force` | boolean | Force running update even if previous update failed
+
+
+### Response
+A `Plugins Update` resource.
+
+## Get Plugins-Update
+
+> Request Example
+
+```shell
+$ curl -X GET \
+    --header "Tenant: <manager-tenant>" \
+    -u <manager-username>:<manager-password> \
+    "<manager-ip>/api/v3.1/plugins-updates/<plugins-update-id>?_include=id"
+```
+
+```python
+# Using CloudifyClient
+plugins_update = client.plugins_update.get(plugins_update_id=plugins_update_id)
+```
+
+> Response Example
+
+```json
+{
+ "forced": false,
+ "blueprint_id": "baddd86fb-864b-483e-a98a-d53e2b728ccd",
+ "tenant_name": "default_tenant",
+ "created_at": "2019-07-10T07:24:40.520Z",
+ "visibility": "tenant",
+ "private_resource": false,
+ "state": "executing_workflow",
+ "temp_blueprint_id": "bbf4b172-3b21-460c-aed8-4f55b55f3b4f",
+ "created_by": "admin",
+ "deployments_to_update": [
+    "dd7b03fb3-8f92-404c-8ddd-d81bd6a2bc9b"
+ ],
+ "resource_availability": "tenant",
+ "id": "ce63a9d6-726b-4ff2-a267-f953af5dc32d",
+ "execution_id": "8ec77e89-b418-4472-9b44-fe9d7a7fe163"
+}
+
+```
+
+`GET "<manager-ip>/api/v3.1/plugins-updates/<plugins-update-id>"`
+
+Get a plugins update. **Supported for Cloudify Manager 5.0 and above.**
+
+### Response
+A `Plugins Update` resource.
+
+## List Plugins Updates
+
+> Request Example
+
+```shell
+$ curl -X GET \
+    --header "Tenant: <manager-tenant>" \
+    -u <manager-username>:<manager-password> \
+    "<manager-ip>/api/v3.1/plugins-updates?_include=id"
+```
+
+```python
+# Using CloudifyClient
+plugins_updates = client.plugins_update.list(
+        sort=sort_by,
+        is_descending=descending,
+        _all_tenants=all_tenants,
+        _search=search,
+        _offset=pagination_offset,
+        _size=pagination_size,
+        plugins_update_id=plugins_update_id
+    )
+```
+
+> Response Example
+
+```json
+{
+  "items": [
+    {
+      "id": "update1",
+      ...
+    },
+    {
+      "id": "update2",
+      ...
+    },
+    {
+      "id": "update3",
+      ...
+    }
+  ],
+  "metadata": {
+    "pagination": {
+      "total": 3,
+      "offset": 0,
+      "size": 0
+    }
+  }
+}
+```
+
+`GET "{manager-ip}/api/v3.1/plugins-updates"`
+
+Lists the plugins updates. **Supported for Cloudify Manager 5.0 and above.**
```

Comment:
What this list includes in short, something like status and etc

Reply:
It's written below in the "Response" header